business-central-parent/business-central-distribution-wars/src/main/shared/swarm/README.txt

```diff
@@ -2,4 +2,4 @@ To start the jar type:
 java -jar <jar-name> -s <configuration-name>
 
 For example:
-java -jar business-central-webapp-swarm.jar -s sample-swarm-config.yml
\ No newline at end of file
+java -jar business-central-thorntail.jar -s sample-thorntail-config.yml
```

Comment:
I think here again it is more universal name to call it `business-central-standalone.jar`